google-cloud-storage/pom.xml

```diff
@@ -172,6 +172,10 @@
       <version>4.5.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+    </dependency>
```

Comment:
this doesn't look necessary. Could you please remove?

Reply:
done